service/eventer/eventer.go

```diff
@@ -63,14 +61,11 @@ func New(config Config) (spec.Eventer, error) {
 		githubConfig := github.DefaultConfig()
 
 		githubConfig.HTTPClient = config.HTTPClient
-		githubConfig.K8sClient = config.K8sClient
```

Comment:
I separated the k8s and github parts. So the eventer does only do the github event magic. No k8s client needed.